iosApp/iosApp/Pages/Map/MapLayerManager.swift

```diff
@@ -88,6 +84,49 @@ class MapLayerManager: IMapLayerManager {
         }
     }
 
+    func updateVisibleLayers(colorScheme: ColorScheme) {
+        let newLayers = generateLayers(colorScheme: colorScheme)
+        let newLayerIds = newLayers.map(\.id)
+        for layerIdentifier in map.allLayerIdentifiers {
+            do {
+                if layerIdentifier.id.contains(StopLayerGenerator.shared.stopLayerId),
```

Comment:
I don't love this, it creates an implicit convention that all stop layers will include `stop-layer` in the ID. However since the transfer and alert layer ids are indexed, I couldn't see an easy way of identifying stop layers without doing some kind of pattern match